magefile.go

```diff
@@ -33,15 +31,9 @@ var (
 	buildDir       = "./build"
 	storageRepoDir = filepath.Join(buildDir, "package-storage")
 	packagePaths   = []string{filepath.Join(storageRepoDir, "packages"), "./dev/packages/example/"}
-	tarGz          = true
 )
 
 func Build() error {
-	packagePathsEnv := os.Getenv("PACKAGE_PATHS")
```

Comment:
It would be nice to support this in the future directly in the registry to pass in an env variable for this.

Reply:
We can add it in the future, but the goal is to use the config file. If you prefer, we can create a new issue to read same configuration from environment variables, but not sure if there is a specific use case. In all cases I'm aware you can depend on the `config.yml` file.